### ISSUE TITLE
storage: support Amazon Drive

### ISSUE THREAD

**Issue by fawick** [upspin/upspin#289]
It would be nice if there were a Storage implementation for Amazon Drive.

For the lack of a better suited project I open the issue for `upspin/upspin` for now, even though the code for the implementation will most likely land in a different repository.

**Comment by robpike** [comment]
Duplicate of #225.

**Comment by adg** [comment]
Is it? I think Amazon Drive is to Google Drive as Amazon S3 is to Google Cloud Storage.

**Comment by fawick** [comment]
@adg correct, Amazon Drive is aimed at individuals and not at enterprises.

Amazon advertises that, for a fixed price ($60 in the US, €70 in some parts of the EU), end-consumers may store unlimited amount of data. In reality, it's probably some arbitrary secret limit that is seen as "normal use". In my experience this limit is higher than 1 TB. Transfer costs are included, so if the data is retrieved more than a few times it seems to be a better deal for personal use. YMMV.
